### PR TITLE
docs: update README, add diagrams

### PR DESCRIPTION
**Pull request by Jane201510**
#### What type of PR is this?
No code modification, add pictures to the readme to enhance the expression

#### What this PR does / why we need it:
Check if the modification effect meets expectations

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: README-zh-CN.md [hunk 1]
```diff
@@ -5,34 +5,51 @@
 # OceanBase 开发者中心 (ODC)
 
 OceanBase 开发者中心（以下简称 ODC）是开源的全场景数据库开发和数据管理协同工具，通过协同开发解决数据库的变更风险管控、数据管理和数据安全问题。
+![image](https://github.com/oceanbase/odc/assets/15030999/0ec084a6-25e7-4f1c-8fb8-70e9f33c730e)
```
Comment (yizhouxw) [inline README-zh-CN.md:8]: please take care the image size, currently 1680 × 1006, which cost too much network transfer while user load github readme page.
I suggest 50% size, reame page occupty less than 40% screen witdh, 800 witdh will be readable for common scenarios.

File: README.md [hunk 2]
```diff
@@ -5,38 +5,45 @@ English | [中文](./README-zh-CN.md)
...
 - Utilizing modern web architecture, ODC provides unparalleled accessibility to your databases, allowing you to perform SQL development tasks on the go, right from your web browser.
 - ODC boasts a comprehensive and user-friendly SQL development environment that encompasses all the features you'd expect from desktop development tools, and even offers support for PL debugging.
+![image](https://github.com/oceanbase/odc/assets/15030999/1d9e32a6-f359-4286-b0be-82cf514c0883)
 
 
 ### Guard Every Change in Your SQL Development Journey 
 
 - Throughout the entire SQL development process, ODC is your steadfast companion, offering risk control at every turn. From visual object management to SQL queries, data editing, and import/export tasks, ODC comes equipped with built-in risk management tools.
 - ODC introduces project-based collaboration and change approval workflows, complemented by SQL validation rules, SQL window guidelines, and risk level identification.
+![image](https://github.com/oceanbase/odc/assets/15030999/6d90509b-91cb-465a-ac75-1604f96c9ffd)
```
Comment (yizhouxw) [inline README.md:24]: the backgroud color may change to white, as this is the only picture with deep color.

and, there exists one red alert, miss blank between 2 words ?
image

File: README.md [hunk 3]
```diff
@@ -5,38 +5,45 @@ English | [中文](./README-zh-CN.md)
...
 - Beyond time-based data handling, ODC supports batch processing based on partitioning, allowing you to efficiently manage large datasets.
 - With ODC's SQL scheduled tasks, completing computational tasks becomes a breeze, rendering traditional CRONTAB systems obsolete.
+![image](https://github.com/oceanbase/odc/assets/15030999/c95b57fd-3d49-4e41-96f9-32f26cb2e2d3)
+
 
 ### Sensitive Data Protection in Every Scenario 
 
 - ODC's data desensitization capabilities extend to both static and dynamic scenarios, ensuring that sensitive information remains safeguarded during schema changes, SQL queries, result set exports, and data extractions.
 - Security administrators can configure sensitive data rules and desensitization algorithms, ensuring that both DBAs and developers are unable to access sensitive data.
+![image](https://github.com/oceanbase/odc/assets/15030999/56606949-e198-4d80-a484-778e78454d65)
```
Comment (yizhouxw) [inline README.md:39]: how about change the layout from top down to left right, seems too thin, while together with other pirctures.

**Review by yizhouxw — COMMENTED** [review]
wow, it's pretty cool, i love diagram than words.

one point for discussion:
how about remove `Features at a Glance` part , as we have better explaination about features now, the readme doc seems too long, shorter better.

**Comment by yhilmare** [comment]
the target branch is main?

**Comment by Jane201510** [comment]
> the target branch is main?

yes

**Comment by yizhouxw** [comment]
> > the target branch is main?
> 
> yes

often we merge to current sprint branch, e.g. dev/4.2.2 now.
however this pr merge int main branch will be okay, lets' fix conflict while merge dev/4.2.3 to main @yhilmare

**Review by yizhouxw — APPROVED** [review]
lgtm

**Pull request merged**